sandman/source/command.cpp

```diff
@@ -77,11 +77,11 @@ static Time s_RebootDelayStartTime;
 
 // Initialize the system.
 //
-// p_Input:	The input device.
+// input:	The input device.
 //
-void CommandInitialize(Input const& p_Input)
+void CommandInitialize(Input const& input)
 {
-	s_Input = &p_Input;
```

Comment:
Won't change this in this pull request because [code style changes for member variables to be camel will be done in a follow-up pull request](https://github.com/shawn-lindberg/sandman/pull/53#pullrequestreview-2241638297).